services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/gdpr/GdprService.kt

```diff
@@ -80,7 +154,13 @@ class OldGdprService(private val applicationService: ApplicationService) : GdprS
     }
 
     @Transactional(readOnly = true)
-    override fun findApplicationsToDelete(userId: String): List<Application> {
+    override fun canDelete(userId: String): Boolean {
+        findApplicationsToDelete(userId)
+        // Will throw an exception if the information can't be deleted
```

Comment:
This comment might be better to be above the `findApplicationsToDelete` or on the same line with it.

Reply:
I moved the comment.